src/Compilers/Core/Portable/Symbols/NullableAnnotation.cs

```diff
@@ -13,13 +13,7 @@ public enum NullableAnnotation : byte
         /// The expression has not been analyzed, or the syntax is
         /// not an expression (such as a statement).
         /// </summary>
-        NotApplicable = 0,
-        /// <summary>
-        /// The expression comes from a library not updated to C# 8
-        /// or nullable was disabled for the definition, and has no
```

Comment:
This case still applies.

Reply:
I felt like the original comment on NotApplicable was actually sufficient (namely "the expression has not been analyzed"), but I can add these as well if you'd prefer.

---
In reply to: [306882877](https://github.com/dotnet/roslyn/pull/37442#discussion_r306882877) [](ancestors = 306882877)